### PR TITLE
Centos hardening

### LINKED ISSUE #3: add centos support
the issue is that in centos machines ssh service called `sshd`

### PR DESCRIPTION
**Pull request by sergey-korolev**
resolves #3

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   notify: restart sshd
 
 - name: remove requiretty from sudoers
-  command: sed -i -e '/^Defaults    requiretty/ d' /etc/sudoers
+  lineinfile: dest=/etc/sudoers regexp="^Defaults.*requiretty" state=absent
```
Comment (akhon) [inline tasks/main.yml]: @sergey-korolev let's tight up regexp to not have `.*`, we can have only spaces and tabs there

**Review by akhon — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (sergey-korolev) [reply tasks/main.yml]: @akhon fixed

**Comment by akhon** [comment]
@mahmoudimus pls take a look!

**Comment by sergey-korolev** [comment]
@mahmoudimus Hi. Did you review this PR? Have you any comments with it? Thanks.

**Comment by akhon** [comment]
@mjallday @mahmoudimus pls review

**Review by mjallday — APPROVED** [review]
👍

**Pull request merged**